contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -571,6 +665,8 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, C
             previous_contract: version.contract,
         });
     }
+    let latest_stage = LATEST_STAGE.load(deps.storage)?;
```

Comment:
One tip: migration code here do not give any information above the version changes. You can extract the code here to `migrations.rs` and wrap the logic under `v{migrated_version}` and call the function here.